openwisp_firmware_upgrader/admin.py

```diff
@@ -133,6 +133,14 @@ def upgrade_selected(self, request, queryset):
         'Mass-upgrade devices related ' 'to the selected build'
     )
 
+    def change_view(self, request, object_id, form_url='', extra_context=None):
+        app_label = self.model._meta.app_label
+        extra_context = extra_context or {}
+        self.change_form_template = 'admin/change_form_overriden.html'
```

Comment:
You mentioned in the chat that:
`change_form_template = 'admin/change_form.html'` in the class is not working for you, right? :smile:

Reply:
It was working for me, however, to make sure that this template is more specific, I changed the dir to `admin/firmware_build/change_form.html'`; let me know if it works for you as well, if not, it's a cache related issue, please check your broswer's cache! :smile: